js/pivx_shield.ts

```diff
@@ -273,6 +282,9 @@ export class PIVXShield {
       shieldData.lastProcessedBlock,
```

Comment:
We need sync from scratch if mapNullifierNote is undefined or old wallets won't have the complete activity